testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.resizing.tests.js

```diff
@@ -15,11 +15,12 @@ QUnit.testStart(function() {
 
         <div id='container'>
             <div id="dataGrid"></div>
-            <div id="dataGridWithStyle" style="width: 500px;"></div>
+            <div id="dataGridWithStyle"></div>
         </div>
     `;
 
     $('#qunit-fixture').html(markup);
+    $('#dataGridWithStyle').css('width', '500px');
```

Comment:
Let's move it to the ```<style>``` CSS rules:

```suggestion
    #dataGridWithStyle {
      width: 500px;
    }
```